theano/gpuarray/basic_ops.py

```diff
@@ -167,6 +168,8 @@ def get_dtype(t):
 
     def _get_c_flags(self):
         res = []
+        if self.flags.get('cflags', '') != '':
```

Comment:
If you don't want to use it, you don't set it at all.  Passing None is not supported.